packages/roosterjs-editor-types/lib/enum/PasteType.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Enum for paste options
+ */
+export const enum PasteType {
+    /**
+     * Simple paste with not content change
```

Comment:
Actually this is not accurate. Content might be changed. We just need to say this is t he default paste behavior and auto select the pasting format.